extensions/wikia/Recirculation/templates/client/rail.mustache

```diff
@@ -1,13 +1,14 @@
-<section class="recirculation-rail module">
-	<h2>{{ title }}</h2>
+<section class="recirculation-rail module group-{{ group }}">
+	<h2>{{{ title }}}</h2>
 	<ul class="thumbnails">
 		{{#items}}
-			<li class="title-thumbnail">
+			<li class="rail-item {{classes}}" data-flag="{{ flag }}" data-index="{{ index }}">
```

Comment:
Try `{{@index}}` (http://stackoverflow.com/questions/5021495/in-mustache-how-to-get-the-index-of-the-current-section). Would remove the need for `addIndex`